packages/react-inbox/src/components/Inbox/index.tsx

```diff
@@ -51,6 +51,9 @@ const StyledTippy = styled(LazyTippy)<{
         maxHeight: 545,
         display: "flex",
         flexDirection: "column",
+        "&, *": {
```

Comment:
oh, I've not seen this before. what does this selection target?

Reply:
myself and all my children